airbyte-integrations/connectors/source-sendgrid/source_sendgrid/streams.py

```diff
@@ -72,9 +73,13 @@ def next_page_token(self, response: requests.Response) -> Optional[Mapping[str,
 class SendgridStreamIncrementalMixin(HttpStream, ABC):
     cursor_field = "created"
 
-    def __init__(self, start_time: int, **kwargs):
+    def __init__(self, start_time: Optional[Union[int, str]], **kwargs):
         super().__init__(**kwargs)
+        self._start_time = start_time or 0
```

Comment:
self._start_time = int(pendulum.parse(self._start_time).timestamp()) if isinstance(self._start_time, str) else start_time or 0

Reply:
No need to re-declare here, I believe.